lib/src/model/model_element.dart

```diff
@@ -427,6 +427,12 @@ abstract class ModelElement
       }
     }
 
+    if (element.nonSynthetic2 case Annotatable(:var metadata2)) {
```

Comment:
I found the `nonSynthetic2` getter.

Perhaps that can also be used above instead of all the dancing around "PropertyinducingElement"